pkgs/desktops/cinnamon/cinnamon-menus/default.nix

```diff
@@ -0,0 +1,24 @@
+{ fetchFromGitHub, glib, gobjectIntrospection, meson, ninja, pkgconfig, stdenv, wrapGAppsHook, libxml2, cmake }:
+
+stdenv.mkDerivation rec {
+  pname = "cinnamon-menus";
+  version = "4.4.0";
+
+  src = fetchFromGitHub {
+    owner = "linuxmint";
+    repo = "${pname}";
+    rev = "${version}";
+    sha256 = "0q4qj28swi2y93fj7pfil68l2cf9gmhbk6jmr8d70l54xf7sigsh";
+  };
+
+  buildInputs = [ glib gobjectIntrospection pkgconfig ];
+  nativeBuildInputs = [ meson ninja wrapGAppsHook cmake ];
+
+  meta = {
+    homepage = "http://cinnamon.linuxmint.com";
```

Comment:
A webpage that's still valid that's similar to this is https://developer.linuxmint.com/projects/cinnamon-projects.html.
Or maybe just use the url to github.